core/src/main/java/io/fabric8/maven/core/util/kubernetes/KubernetesResourceUtil.java

```diff
@@ -191,6 +191,15 @@ public boolean accept(File dir, String name) {
 
     protected final static void initializeKindFilenameMapper() {
         final Map<String, List<String>> mappings = KindFilenameMapperUtil.loadMappings();
+        updateKindFilenameMapper(mappings);
+    }
+
+    protected final static void remove(String kind, String filename) {
```

Comment:
not sure where this is use, but could miss it. If not necessary I would avoid it, so making the mappings read-only.

Reply:
It is protected because it is needed in a test, since it is static class we need to have a way to clean it after run the test.